scripts/generate_python_docs.sh

```diff
@@ -32,7 +32,7 @@ run_pydocgen() {
   pipenv --python 3
   pipenv install
   for pkg in "${PACKAGES[@]}"; do
-    pipenv run pip install --pre "${pkg}"
+    pipenv run pip install "${pkg}"
```

Comment:
Removed `--pre` so we only install the released packages for generating the Python API docs.